### PR TITLE
[New post - publish after 10/26 only] Generative AI for Go developers - Build a Serverless image generation solution using Amazon Bedrock

### PR DESCRIPTION
**Pull request by abhirockzz**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [x] The title of this pull request reflects the title of my content.
- [x] I have reviewed the [Content Review Checklist](https://github.com/build-on-aws/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [x] This pull request contains one piece of content (post or page).

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Review by mikegc-aws — COMMENTED** [review]
Provided comments in Slack.

**Review by cherbk — APPROVED** [review]

File: tutorials/amazon-bedrock-lambda-image-gen-golang/index.md [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+
+type Artifact struct {
+	Base64       string `json:"base64"`
+	FinishReason string `json:"finishReason"`
+}
+```
+
+## Conclusion
+
+In this tutorial, you used AWS CDK to deploy a serverless image generation solution that was implemented using Amazon Bedrock and AWS Lambda, and accessed using a static website on S3 via a CloudFront domain.
```
Comment (DalM2023) [inline tutorials/amazon-bedrock-lambda-image-gen-golang/index.md:320]: We want to invite the user to stay in the blog repository longer. Please add a sentence that is apropos of your topic and sends them to 2-3 blogs. 

EX: "To read more/learn more about X, visit (LINK) and (LINK)."

File: tutorials/amazon-bedrock-lambda-image-gen-golang/index.md [hunk 2]
```diff
@@ -0,0 +1,322 @@
+---
+title: "Generative AI for Go developers - Build a Serverless image generation solution using Amazon Bedrock"  
+description: "This tutorial will walk you through how to use AWS CDK to deploy a Serverless Go application implemented using Amazon Bedrock and AWS Lambda, and accessed as a static website on Amazon S3 via a Amazon CloudFront domain."
+tags:
```
Comment (DalM2023) [inline tutorials/amazon-bedrock-lambda-image-gen-golang/index.md:4]: We can only use 5 tags. Please choose the best ones.

File: tutorials/amazon-bedrock-lambda-image-gen-golang/index.md [hunk 3]
```diff
@@ -0,0 +1,322 @@
+---
+title: "Generative AI for Go developers - Build a Serverless image generation solution using Amazon Bedrock"  
+description: "This tutorial will walk you through how to use AWS CDK to deploy a Serverless Go application implemented using Amazon Bedrock and AWS Lambda, and accessed as a static website on Amazon S3 via a Amazon CloudFront domain."
```
Comment (DalM2023) [inline tutorials/amazon-bedrock-lambda-image-gen-golang/index.md:3]: The descriptors need to be some 60 characters long. They can't repeat the messaging in the title or use the same words. It needs to focus on what solution the reader gets from reading the piece. Please rewrite.

EX: "Use third-party model providers to XYZ. "

**Review by DalM2023 — COMMENTED** [review]
1st round edit done. Ready for review.

File: tutorials/amazon-bedrock-lambda-image-gen-golang/index.md [hunk 4]
```diff
@@ -0,0 +1,322 @@
...
+| 💻 Code Sample         | Code sample used in tutorial on [GitHub](https://github.com/build-on-aws/amazon-bedrock-lambda-image-generation-golang)                             |
+| 📢 Feedback            | <a href="https://pulse.buildon.aws/survey/DEM0H5VW" target="_blank">Any feedback, issues, or just a</a> 👍 / 👎 ?    |
+| ⏰ Last Updated     | 2023-10-27  
+
+## Pre-requisites
+
+Before starting this tutorial, you will need the following:
+
+- An AWS Account (if you don't yet have one, you can create one and [set up your environment here](https://aws.amazon.com/getting-started/guides/setup-environment/?sc_channel=el&sc_campaign=genaiwave&sc_content=amazon-bedrock-lambda-image-gen-website&sc_geo=mult&sc_country=mult&sc_outcome=acq)).
+- [Go programming language](https://go.dev/dl/) (**v1.19** or higher).
```
[the remark this answers is not in the document]
Comment (abhirockzz) [reply tutorials/amazon-bedrock-lambda-image-gen-golang/index.md:50]: No, its not a proper name. The words "programming language" is used in a generic context - should not be capitalised.

File: tutorials/amazon-bedrock-lambda-image-gen-golang/index.md [hunk 5]
```diff
@@ -0,0 +1,322 @@
+---
```
[the remark this answers is not in the document]
Comment (abhirockzz) [reply tutorials/amazon-bedrock-lambda-image-gen-golang/index.md:1]: Thanks! I've updated content per feedback

**Review by damonk10 — APPROVED** [review]

**Pull request merged**